### DISCUSSION TITLE
BIND-Like $generate Support

### DISCUSSION THREAD

**Discussion by sjkra** [ngoduykhanh/PowerDNS-Admin#1244] [Q&A] [answered]
I see that pdns can use $GENERATE from bind configs, and it is working when I importuned my zones from my bind configs. 
Is there a way to input this in PowerDNS-Admin?

I did try 
`$GENERATE 225-245/1 private64-$ A 10.112.64$` 
in a A record and I got a parsing error.

**Comment by jbe-dw** [comment, marked as answer]
Hello,

It is not possible so far.

Regards

**Reply by sjkra** [reply]
is it on the roadmap?

**Reply by AzorianMatt** [reply]
@sjkra No currently but I will look into what it would take. It most likely will not make it into the legacy edition of the application, but I see no reason it can't be added to the road map for the next-generation edition.
